UDPClientGUI/MyUDPClient.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UDPCommGUI
+{
+    class MyUDPClient : UdpClient, IClientInterface
+    {
+
+        private string connectionIP;
+        private int connectionPort;
+        private string inputMessage;
+
+
+        public string ConnectionIP
+        {
+            get { return connectionIP; }
+            set { connectionIP = value; }
+        }
```

Comment:
With Autoproperties, if you're not doing anything at the time of getting/setting, you don't need to have it this way.

`public string ConnectionIP { get; set; }`

Where you would use a backing field (like it's done here), is if it's a different data type to the Property or if you have extra logic around getting/setting.

Example:
```
private int myVariable;

public string MyVariable => myVariable.ToString();
```
While you're at it, check out `Expression Bodied Members`

Reply:
Changed as suggested.